.github/workflows/package-debian.yml

```
@@ -64,19 +66,45 @@ jobs:
       - name: Build
         id: build
         env:
-          SILKIT_SOURCE_URL: ${{ github.workspace }}/sil-kit
-          SILKIT_PKG_URL: ${{ github.workspace }}/sil-kit-pkg
           DEBFULLNAME: ${{ inputs.debian_fullname }}
           DEBEMAIL: ${{ inputs.debian_fullname }}
           CI_RUN: "1"
+          BUILD_SETTINGS: |
+              {
+
+                  "SilKitInfo": {
+                      "url": "sil-kit",
+                      "ref": "",
+                      "recursive": true,
+                      "is_local": true
+                  },
+                  "package_repo_path": "sil-kit-pkg",
+                  "version": {
+                      "major": 4,
+                      "minor": 0,
+                      "patch": 53,
+                      "suffix": "rc-1"
+                  },
+                  "work_dir": "${{ env.PKG_WORKDIR }}",
+                  "keep_temp": true,
+                  "output_dir": "${{ env.PKG_OUTDIR }}",
+                  "platform": "Ubuntu-20.04"
+              }
+
         run: |
-          sh sil-kit-pkg/.github/actions/build_deb.sh 20.04
+          mkdir -p $GITHUB_WORKSPACE/$PKG_WORKDIR
+          mkdir -p $GITHUB_WORKSPACE/$PKG_OUTDIR
+          touch $PWD/$PKG_WORKDIR/build_cfg.json
+          printf %s $BUILD_SETTINGS  > $PKG_WORKDIR/build_cfg.json
```

is it necessary to use absolute paths? aren't we located in `$GITHUB_WORKSPACE` anyway?
for example:
```suggestion
          mkdir -p $PKG_WORKDIR
          mkdir -p $PKG_OUTDIR
          touch $PKG_WORKDIR/build_cfg.json
          printf %s $BUILD_SETTINGS  > $PKG_WORKDIR/build_cfg.json
```

---

I had some trouble with docker builds and workspace with multiple checkouts (sil-kit-pkg and sil-kit). But this could just have been me misunderstanding GITHUB_WORKSPACE and github.workspace which are slightly different. I can revisit the PATH situation again to see if we can be less verbose